recipe_scrapers/_abstract.py

```diff
@@ -88,6 +104,43 @@ def image(self):
         except AttributeError:  # if image not found
             raise NotImplementedError("This should be implemented.")
 
+    @Decorators.bcp47_validate
+    @Decorators.schema_org_priority
+    def language(self):
+        """
+        Human language the recipe is written in.
+
+        May be overridden by individual scrapers.
+        """
+        candidate_languages = set()
+        html = self.soup.find(
+            'html',
+            {'lang': True}
+        )
+        candidate_languages.add(html.get('lang'))
+
+        # Deprecated: check for a meta http-equiv header
+        # See: https://www.w3.org/International/questions/qa-http-and-lang
+        meta_language = self.soup.find(
```

Comment:
Honestly, I'd ignore the meta tag (as it's deprecated as you've said). My quick run-through the sites supported showed that only mindmegette.hu is using it. I think it's a small gain on our end to support deprecated features.

Reply:
Mostly agree; this is a slightly selfish implementation, because I'd like mindmegette recipes to be marked as `hu-HU` by the system I'm building.  I could add a special case on the client-side.. let me consider it a little more.

Reply:
Going to stick with the BCP-47 validation approach for now.  If we see any negative impact as a result then I'll reconsider.